res2df/equil.py

```diff
@@ -283,7 +283,7 @@ def fill_parser(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
         parser (argparse.ArgumentParser or argparse.subparser): parser to
             fill with arguments
     """
-    parser.add_argument("DATAFILE", help="Name of Eclipse DATA file.")
+    parser.add_argument("DATAFILE", help="Name of reservoir DATA file.")
```

Comment:
So the user sees the argument name `DATAFILE` so there is no need to restate it in the `help` text. I think it would be more helpful to have the text say `Name of the .DATA input file for a reservoir simulator`, or similar.